project/Dependencies.scala

```diff
@@ -1,28 +1,43 @@
 import sbt._
 
 object Dependencies {
-  val minitestVersion = "2.2.2"
-  val circeVersion = "0.10.0"
-  val http4sVersion = "0.20.0-M4"
-  val grpcNettyVersion = "1.18.0"
-  val cirisVersion = "0.12.1"
-  val doobieVersion = "0.7.0-M2"
+  val circeVersion          = "0.10.0"
+  val doobieVersion         = "0.7.0-M2"
+  val http4sVersion         = "0.20.0-M5"
+  val grpcNettyVersion      = "1.18.0"
+  val liquibaseVersion      = "3.5.3"
+  val minitestVersion       = "2.2.2"
+  val postgresDriverVersion = "42.2.4"
+  val samV                  = "0.1-37bc074"
+  val scalaTestVersion      = "3.0.5"
+  val scalikejdbcVersion    = "3.3.2"
 
   val common = List(
-    "io.grpc" % "grpc-netty" % grpcNettyVersion,
-    "io.monix" %% "minitest" % minitestVersion % "test",
-    "io.monix" %% "minitest-laws" % minitestVersion % "test",
-    "io.chrisdavenport" %% "log4cats-slf4j"   % "0.2.0",
-    "io.circe" %% "circe-core" % circeVersion,
-    "io.circe" %% "circe-parser" % circeVersion,
-    "org.http4s" %% "http4s-circe" % http4sVersion,
-    "org.http4s" %% "http4s-blaze-client" % http4sVersion,
-    "com.github.pureconfig" %% "pureconfig" % "0.10.1",
-    "io.sentry" % "sentry-logback" % "1.7.16", // see doc https://docs.sentry.io/clients/java/modules/logback/
-    "org.tpolecat" %% "doobie-core" % doobieVersion,
-    "org.tpolecat" %% "doobie-postgres"  % doobieVersion,
-    "org.tpolecat" %% "doobie-hikari"    % doobieVersion,
-    "org.tpolecat" %% "doobie-specs2"   % doobieVersion % "test"
+    "com.github.pureconfig"             %% "pureconfig"          % "0.10.1",
+    "com.iheart"                        %% "ficus"               % "1.4.3",
+    "com.zaxxer"                        %  "HikariCP"            % "1.3.+",
+    "io.chrisdavenport"                 %% "log4cats-slf4j"      % "0.2.0",
+    "io.circe"                          %% "circe-core"          % circeVersion,
+    "io.circe"                          %% "circe-generic"       % circeVersion,
+    "io.circe"                          %% "circe-parser"        % circeVersion,
+    "io.grpc"                           %  "grpc-netty"          % grpcNettyVersion,
+    "io.monix"                          %% "minitest"            % minitestVersion    % "test",
+    "io.monix"                          %% "minitest-laws"       % minitestVersion    % "test",
+    "io.sentry"                         %  "sentry-logback"      % "1.7.16",          // see doc https://docs.sentry.io/clients/java/modules/logback/
+    "org.broadinstitute.dsde.workbench" %% "sam-client"          % samV,
+    "org.http4s"                        %% "http4s-blaze-client" % http4sVersion,
+    "org.http4s"                        %% "http4s-blaze-server" % http4sVersion,
```

Comment:
can you move this out of common...core module doesn't need `http4s-blaze-server`, neither automation